tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -149,6 +150,13 @@ def keyboard(self):
         from gaiatest.apps.keyboard.app import Keyboard
         return Keyboard(self.marionette)
 
+    @property
+    def manifest_url(self):
+        return '{}{}.{}/manifest.webapp'.format(self.DEFAULT_PROTOCOL, format(self.__class__).lower(), self.DEFAULT_APP_HOSTNAME)
```

Comment:
For more details, see https://docs.python.org/2.7/library/stdtypes.html?highlight=__class__#instance.__class__